### PR TITLE
[tests] [by Artem] Enabled PCH for testing builds that have COMGR enabled (removed WORKAROUND_ISSUE_898)

### PR DESCRIPTION
**Pull request by junliume**
Reinstate #1478 after blocking issue has been fixed (#1481 and #1419)

### CODE REVIEW

**Review by atamazov — DISMISSED** [review]
LGTM!

**Comment by junliume** [comment]
@JehandadKhan  need another approval to merge

**Comment by junliume** [comment]
Gentle ping for @JehandadKhan @asroy @zjing14 wo need this to fix some existing issues. Urgency is high.

**Comment by junliume** [comment]
**Update all ROCm 5.2 targeted PR/issues as blockers.**
Closely monitoring CI, and this one should be able to land in timeframe.

**Review by JehandadKhan — APPROVED** [review]

File: src/kernels/gpu_general_tensor_reorder_kernel/general_tensor_reorder_kernel_util.hpp [hunk 1]
```diff
@@ -24,7 +24,19 @@
  *
  *******************************************************************************/
 #ifndef GENERAL_TENSOR_REORDER_UTIL_HPP
-#include <cstdint>
+#ifdef __HIPCC_RTC__
```
Comment (atamazov) [inline src/kernels/gpu_general_tensor_reorder_kernel/general_tensor_reorder_kernel_util.hpp:27]: @junliume Thanks for catching this!

**Pull request merged**